<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiltersDockWidget</class>
 <widget class="QDockWidget" name="FiltersDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>233</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>233</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>524287</height>
   </size>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Filters</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnModifyFilterPreset">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="BlastToolBar.qrc">
          <normaloff>:/AppMainWindow/images/EditWrench.png</normaloff>:/AppMainWindow/images/EditWrench.png</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFilterPreset">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Filter Preset</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnAddFilterPrest">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="BlastToolBar.qrc">
          <normaloff>:/AppMainWindow/images/Add.png</normaloff>:/AppMainWindow/images/Add.png</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="btnRemoveFilterPreset">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="BlastToolBar.qrc">
          <normaloff>:/AppMainWindow/images/Remove_icon.png</normaloff>:/AppMainWindow/images/Remove_icon.png</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="btnSaveFilterPreset">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="BlastToolBar.qrc">
          <normaloff>:/AppMainWindow/images/saveDisc.png</normaloff>:/AppMainWindow/images/saveDisc.png</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxFilterPreset">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="6">
       <widget class="QMenuBar" name="menuBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <widget class="QMenu" name="menuRule">
         <property name="title">
          <string>Rule</string>
         </property>
         <addaction name="actionAllDescendants"/>
         <addaction name="actionAllParents"/>
        </widget>
        <widget class="QMenu" name="menuDepth">
         <property name="title">
          <string>Depth</string>
         </property>
         <addaction name="actionDepthAll"/>
         <addaction name="actionDepth0"/>
         <addaction name="actionDepth1"/>
         <addaction name="actionDepth2"/>
         <addaction name="actionDepth3"/>
         <addaction name="actionDepth4"/>
         <addaction name="actionDepth5"/>
        </widget>
        <widget class="QMenu" name="menuItem_Type">
         <property name="title">
          <string>Item Type</string>
         </property>
         <addaction name="actionItemTypeAll"/>
         <addaction name="actionChunk"/>
         <addaction name="actionSupportChunk"/>
         <addaction name="actionStaticSupportChunk"/>
         <addaction name="actionBond"/>
         <addaction name="actionWorldBond"/>
        </widget>
        <widget class="QMenu" name="menuOperator">
         <property name="title">
          <string>Operator</string>
         </property>
         <addaction name="actionEqualTo"/>
         <addaction name="actionNotEquaTo"/>
        </widget>
        <addaction name="menuRule"/>
        <addaction name="menuDepth"/>
        <addaction name="menuItem_Type"/>
        <addaction name="menuOperator"/>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>6</number>
      </property>
      <item row="2" column="0" colspan="3">
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelect">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="BlastToolBar.qrc">
            <normaloff>:/AppMainWindow/images/Select.png</normaloff>:/AppMainWindow/images/Select.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnVisible">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="BlastToolBar.qrc">
            <normaloff>:/AppMainWindow/images/visibilityToggle_visible.png</normaloff>:/AppMainWindow/images/visibilityToggle_visible.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInVisible">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="BlastToolBar.qrc">
            <normaloff>:/AppMainWindow/images/visibilityToggle_notVisible.png</normaloff>:/AppMainWindow/images/visibilityToggle_notVisible.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionAllDescendants">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All Descendants</string>
   </property>
  </action>
  <action name="actionAllParents">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All Parents</string>
   </property>
  </action>
  <action name="actionDepthAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionDepth0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Depth 0</string>
   </property>
  </action>
  <action name="actionDepth1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Depth 1</string>
   </property>
  </action>
  <action name="actionDepth2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Depth 2</string>
   </property>
  </action>
  <action name="actionDepth3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Depth 3</string>
   </property>
  </action>
  <action name="actionDepth4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Depth 4</string>
   </property>
  </action>
  <action name="actionDepth5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Depth 5</string>
   </property>
  </action>
  <action name="actionItemTypeAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionChunk">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chunk</string>
   </property>
  </action>
  <action name="actionSupportChunk">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Support Chunk</string>
   </property>
  </action>
  <action name="actionStaticSupportChunk">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Static Support Chunk</string>
   </property>
  </action>
  <action name="actionBond">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bond</string>
   </property>
  </action>
  <action name="actionWorldBond">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>World Bond</string>
   </property>
  </action>
  <action name="actionEqualTo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Equal To</string>
   </property>
  </action>
  <action name="actionNotEquaTo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Not Equal To</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>comboBoxFilterPreset</tabstop>
  <tabstop>btnAddFilterPrest</tabstop>
  <tabstop>btnModifyFilterPreset</tabstop>
  <tabstop>btnRemoveFilterPreset</tabstop>
  <tabstop>btnSaveFilterPreset</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>btnSelect</tabstop>
  <tabstop>btnVisible</tabstop>
  <tabstop>btnInVisible</tabstop>
 </tabstops>
 <resources>
  <include location="BlastToolBar.qrc"/>
 </resources>
 <connections/>
</ui>
